src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -1298,18 +1298,21 @@ private bool W3CIdFlagsSet
             get => (_w3CIdFlags & ActivityTraceFlagsIsSet) != 0;
         }
 
-        private bool IsFinished
+        /// <summary>
+        /// Indicates whether this Activity object is stopped
```

Comment:
Maybe document that this is already true in the ActivityStopped listener (assuming and hoping this is the case). 
Could also be added to the unit test.